src/service.ts

```diff
@@ -168,11 +173,16 @@ export class PisaService extends StartStopService {
     private appointment(tower: PisaTower) {
         return async (req: express.Request, res: express.Response, next: express.NextFunction) => {
             try {
-                const appointment = await tower.addAppointment(req.body);
+                const { appointment, signature } = await tower.addAppointment(req.body);
 
                 // return the appointment
                 res.status(200);
-                res.send(appointment);
+
+                // with signature
+                res.send({
+                    appointment,
+                    signatures: [signature]
```

Comment:
why plural signature and an array?

Reply:
While we only have one now, aren't we going to give multiple signatures in the receipt?

Reply:
Maybe there's some confusion coming from the fact that this class is called PisaService, when actually if we look at what it does perhaps a better name would be PisaTowerService. A pisa tower should probably only have one signing key, then we can aggregate those signatures as part of some other component.

Reply:
Currently, PisaService hosts the `/appointment` endpoint, which is the user/customer's interface, so I think it makes sense to leave its interface (and response) to what the user expects. When we build a (Pisa)TowerService, then PisaService can communicate with that instead. So I think the component that returns one signature should be the (yet missing) one, not the api endpoint.

Reply:
Done, put directly the signature without the array.